src/nova-components/CreateFavorites/resources/js/components/TextInput.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+  <div class="card">
+    <div class="flex border-b border-40">
+      <div class="w-1/5 px-8 py-6">
+        <label for="title" class="inline-block text-80 pt-2 leading-tight">
+          {{ label }}
+          <span class="text-danger text-sm">*</span></label
+        >
+      </div>
+      <div class="py-6 px-8 w-1/2">
+        <input
+          dusk="title"
+          list="title-list"
+          type="text"
+          :placeholder="placeholder"
+          class="w-full form-control form-input form-input-bordered"
+          :class="{ 'border-danger': error }"
+          v-bind:value="input"
+          v-on:input="$emit('input', $event.target.value)"
```

Comment:
don't use events to pass data to parent. hint: pass a function to the component as a prop